docs/src/examples/components/List/Types/ListExampleSelected.knobs.tsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import Knobs from 'docs/src/components/Knobs/Knobs'
+
+const ListExampleSelectedKnobs: any = props => {
+  const { onKnobChange, selected } = props
+
+  return (
+    <Knobs>
+      <Knobs.Boolean name="selected" value={selected} onChange={onKnobChange} />
+    </Knobs>
+  )
+}
+
+ListExampleSelectedKnobs.propTypes = {
+  onKnobChange: PropTypes.func.isRequired,
+  selected: PropTypes.bool,
```

Comment:
nit: we have `active` prop for it being used throughout the library (e.g. `MenuItem`), lets make the name consistent

Reply:
You are correct. I was not sure if "active" would be the best word to use. I thought for ```MenuItem``` it makes sense but a can a list item be "active". Anyway, I will make the change. 👍